### PR TITLE
New Customisation Feature

### PR DESCRIPTION
**Pull request by AeneasChristodoulou**
adding customisation for the GUI through "own_search_entries/entries/main.py"

also, fixed one Typo in a Comment^^

### CODE REVIEW

File: python_search/config.py [hunk 1]
```diff
@@ -85,7 +87,7 @@ class ConfigurationLoader:
     Loads the application from the environment.py
     """
 
-    def load(self) -> PythonSearchConfiguration:
+    def load_entries(self) -> PythonSearchConfiguration:
```
Comment (jeanCarloMachado) [inline python_search/config.py]: Could you also remove the return type? it is wrong.

**Review by jeanCarloMachado — APPROVED** [review]
LGTM, thank you for the contributions!

File: python_search/config.py (hunk 1 above)
Comment (AeneasChristodoulou) [reply python_search/config.py]: alright, done

**Comment by AeneasChristodoulou** [comment]
added changelog update

**Pull request merged**